exercises/concept/remote-control-cleanup/.docs/introduction.md

```diff
@@ -17,4 +17,18 @@ var inner = new Outer.CInner();
 
 You can set access levels for inner types.
 
-Private members of the outer type are in scope for members of the inner type but not vice versa.
+In order to access members of the containing type simply hand over an containing type to the inner type's constructor. Private members of the outer type are in scope for members of the inner type but not vice versa.
+
+```csharp
+public class Outer
+{
+    public class CInner
+    {
+        private Outer parent;
+
+        public CInner(Outer parent) {
+            this.parent = parent
+        }
+    }
+}
+```
```

Comment:
I'm not sure about this change. It feels a bit like overexplaining things to me, whereas we prefer to have students figure things out for themselves where possible.

Reply:
Hi Erik. Thanks for your feedback. My understanding is that the Introduction section should provide some theory necessary for the exercise. Is that correct? I found the theory a bit lacking which is why I added the explanation. Would you be okay with adding a link to the C# documentation instead?

Reply:
@ErikSchierboom I reverted this addition